storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -275,6 +276,37 @@ func (s *commitLogSource) shouldEncodeSeries(
 	return ranges.Overlaps(blockRange)
 }
 
+func (s *commitLogSource) shouldIncludeInIndex(
+	shard uint32,
+	ts time.Time,
```

Comment:
Any reason why this is `ts time.Time` vs `dp ts.Datapoint` in the other method?  Can we make these symmetrical, either one or the other?

Reply:
sure I just figured I'd only pass the required information. Will update the other method.